Source/Krypton Components/Krypton.Toolkit/General/PlatformInvoke.cs

```diff
@@ -23,6 +23,15 @@
 #pragma warning disable 649
 
 
+using System;
```

Comment:
Why are these in here, if there is supposed to be a removal taking place ?

Reply:
The problem here is that this particular class gets confused on what resources to access. Unsure why, as they're not needed.